examples/js/loaders/AMFLoader.js

```diff
@@ -48,18 +48,18 @@ THREE.AMFLoader.prototype = {
 
 	parse: function ( data ) {
 
-		function loadDocument( data ) {
+		function _loadDocument( data ) {
```

Comment:
What @mrdoob said. : - )

Are you specifying flat shading because you do not support normals yet?